frontend/src/routes/(authenticated)/user/+page.svelte

```diff
@@ -37,31 +42,97 @@
       );
     }
   }
+
+  let sendingVerificationEmail = false;
+  let sentVerificationEmail = false;
+
+  async function sendVerificationEmail(): Promise<void> {
+    sendingVerificationEmail = true;
+    const result = await fetch('/api/user/sendVerificationEmail', { method: 'POST' });
+    if (!result.ok) throw Error(`Failed to send verification email. ${result.status}: ${result.statusText}.`);
+    sentVerificationEmail = true;
+    sendingVerificationEmail = false;
+  }
 </script>
 
 <Page>
-  <div class="content-center">
-    <Form {enhance}>
-      <Input
-        id="name"
-        label={$t('account_settings.name')}
-        type="text"
-        error={$errors.name}
-        bind:value={$form.name}
-        autofocus
-      />
-      <Input
-        id="email"
-        label={$t('account_settings.email')}
-        type="email"
-        error={$errors.email}
-        bind:value={$form.email}
-      />
-      <a class="link my-4" href="/resetPassword">
-        {$t('account_settings.reset_password')}
-      </a>
-      <FormError error={$message} />
-      <Button loading={$submitting}>{$t('account_settings.button_update')}</Button>
-    </Form>
-  </div>
+  <svelte:fragment slot="header">
+    {$t('account_settings.title')}
+  </svelte:fragment>
+  {#if data.verifiedEmail || data.changedEmail}
```

Comment:
man this is a lot. could we move it into a component to simplify this page a bit?